cdap-data-fabric/src/main/java/co/cask/cdap/store/StoreDefinition.java

```diff
@@ -37,79 +37,37 @@ private StoreDefinition() {
   }
 
   /**
-   * Create all system tables.
+   * Create all system tables. A boolean flag can be used to skip creating tables that already exist.
    *
    * @param tableAdmin the table admin to create the table
+   * @param registry the registry to get initialized
+   * @param overWrite true to create tables no matter the tables exist or not, false to skip tables that already exist
    */
   public static void createAllTables(StructuredTableAdmin tableAdmin, StructuredTableRegistry registry,
                                      boolean overWrite) throws IOException, TableAlreadyExistsException {
```

Comment:
`overWrite` is unused here although `DefaultPreviewRunner` calls this method with overwrite true.